src/main/java/com/google/sps/servlets/ServletHelper.java

```diff
@@ -43,8 +46,7 @@ public static String getApiKey(String projectId, String secretId, String version
     }
   }
 
-  public static JSONObject readFromApiUrl(URL url, HttpServletResponse response)
-      throws IOException {
+  public static JSONObject readFromApiUrl(URL url) throws IOException {
```

Comment:
Why did you choose to remove the response from the parameter list -- how are you alerting the user of back-end errors?

Reply:
I removed it because it wasn't being used haha but I agree it should be used to return error messages